packages/server/src/channels/api.ts

```diff
@@ -8,16 +8,26 @@ export class ChannelApi {
 
   async setup() {
     const logger = this.app.logger.root.sub('channels')
-    const webhookRouter = Router()
+    const routers: { [version: string]: { router: Router; path: string } } = {
+      '0.1.0': {
+        router: Router(),
+        path: ''
+      },
+      '1.0.0': {
+        router: Router(),
+        path: '/v1'
+      }
+    }
 
     for (const channel of this.app.channels.list()) {
-      await channel.setup(webhookRouter, logger.sub(channel.meta.name))
+      const router = routers[channel.meta.version]
+      await channel.setup(router.router, logger.sub(channel.meta.name))
 
       channel.logger = this.app.logger.root.sub(channel.meta.name)
```

Comment:
Channes with version 1.0.0 will have a /v1 in the webhook path while the old channels will use the path without /v1.